server/app/models/host_node.rb

```diff
@@ -105,24 +105,6 @@ def rpc_client(timeout = 300)
     RpcClient.new(self.node_id, timeout)
   end
 
-  # @return [Integer]
-  def schedule_counter
-    @schedule_counter ||= 0
-  end
```

Comment:
Specs were failing because some of them were still passing `HostNode` objects to the scheduler, not `Scheduler::Node`s. They now failed after removing this method.